src/TimeSteppers/velocity_and_tracer_tendencies.jl

```diff
@@ -19,23 +19,23 @@ tracer fields, and precalculated diffusivities where applicable. `F` is a named
 forcing functions, `pHY′` is the hydrostatic pressure anomaly.
 
 `parameters` is a `NamedTuple` of scalar parameters for user-defined forcing functions 
-and `time` is the physical time of the model.
+and `clock` is the physical clock of the model.
 """
 @inline function u_velocity_tendency(i, j, k, grid, coriolis, surface_waves, 
-                                     closure, U, C, K, F, pHY′, parameters, time)
+                                     closure, U, C, K, F, pHY′, clock)
 
     return ( - div_ũu(i, j, k, grid, U)
              - x_f_cross_U(i, j, k, grid, coriolis, U)
              - ∂xᶠᵃᵃ(i, j, k, grid, pHY′)
              + ∂ⱼ_2ν_Σ₁ⱼ(i, j, k, grid, closure, U, K)
-             + x_curl_Uˢ_cross_U(i, j, k, grid, surface_waves, U, time)
-             + ∂t_uˢ(i, j, k, grid, surface_waves, time)
-             + F.u(i, j, k, grid, time, U, C, parameters))
+             + x_curl_Uˢ_cross_U(i, j, k, grid, surface_waves, U, clock.time)
+             + ∂t_uˢ(i, j, k, grid, surface_waves, clock.time)
+             + F.u(i, j, k, grid, clock, (velocities=U, tracers=C, diffusivities=K)))
```

Comment:
Should double check to make sure this tuple construction doesn't degrade performance. I doubt it will but should always double check performance benchmarks before tagging a release...

Reply:
For sure, it’s good to check. This stuff gets inlined so my hypothesis is that it compiles to the same thing.